### ISSUE TITLE
Missing PM members

### ISSUE THREAD

**Issue by frankleonrose** [drewfish/arduino-ZeroRegs#3]
Under both Arduino 1.8.1 and PlatformIO 3.6.0a10 I'm getting errors attempting to include ZeroRegs in a sketch for Adafruit Feather M0.

Specifically, the compiler chokes on
- NVMCTRL->PARAM.bit.RWWEEP
- PM->EXTCTRL
- PM->APBCMASK.bit.AC1_
- PM->APBCMASK.bit.LINCTRL_

What am I doing wrong? I haven't been able to discover what might have changed. Were those parameters in the board support package and now they are not? If so, they should be removed here.

```
~/Documents/Arduino/libraries/ZeroRegs/src/ZeroRegs.cpp: In function 'void printZeroRegNVMCTRL(ZeroRegOptions&)':

~/Documents/Arduino/libraries/ZeroRegs/src/ZeroRegs.cpp:481:39: 
error: 'volatile struct NVMCTRL_PARAM_Type::<anonymous>' has no member named 'RWWEEP'
     opts.ser.print(NVMCTRL->PARAM.bit.RWWEEP);
                                       ^
~/Documents/Arduino/libraries/ZeroRegs/src/ZeroRegs.cpp: In function 'void printZeroRegPM(ZeroRegOptions&)':
~/Documents/Arduino/libraries/ZeroRegs/src/ZeroRegs.cpp:622:19: 
error: 'struct Pm' has no member named 'EXTCTRL'
     PRINTFLAG(PM->EXTCTRL, SETDIS);
                   ^
~/Documents/Arduino/libraries/ZeroRegs/src/ZeroRegs.cpp:34:33: note: in definition of macro 'PRINTFLAG'
 #define PRINTFLAG(x,y) do { if (x.bit.y) { opts.ser.print(" " #y); } } while(0)
                                 ^
~/Documents/Arduino/libraries/ZeroRegs/src/ZeroRegs.cpp:692:26: 
error: 'volatile struct PM_APBCMASK_Type::<anonymous>' has no member named 'AC1_'
     if (PM->APBCMASK.bit.AC1_)      { opts.ser.print(" CLK_AC1_APB"); }
                          ^
/Users/frank/Documents/Arduino/libraries/ZeroRegs/src/ZeroRegs.cpp:693:26: 
error: 'volatile struct PM_APBCMASK_Type::<anonymous>' has no member named 'LINCTRL_'
     if (PM->APBCMASK.bit.LINCTRL_)  { opts.ser.print(" CLK_LINCTRL_APB"); }
```

**Comment by Ulli2k** [comment]
Same issue to me.
Everything is up to date. (PlattformIO and ZeroRegs Library)

**Comment by Ulli2k** [comment]
Looks like these are Bits which are not available at the SAMD21G18A?

**Comment by drewfish** [comment]
Sorry for the delay, I'm just taking a look now. I'm able to replicate the same issue.

**Comment by drewfish** [comment]
I think I fixed this in e9ee82c, published as version 0.2.2. Could you try again?

**Comment by Ulli2k** [comment]
works!
perfect thanks!
